examples/ipvs/kubernetes-msi.json

```diff
@@ -0,0 +1,37 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "orchestratorRelease": "1.7",
+      "kubernetesConfig": {
+        "useManagedIdentity": true,
+        "kubeProxyMode" : "ipvs"

```

Comment:
let's add this new field to docs (and possibly instead of having a separate example)